--- a/src/3rdparty/chromium/third_party/breakpad/breakpad/src/config.h.in
+++ b/src/3rdparty/chromium/third_party/breakpad/breakpad/src/config.h.in
@@ -95,6 +95,7 @@
 
 /* Number of bits in a file offset, on hosts where this is settable. */
 #undef _FILE_OFFSET_BITS
+#undef _TIME_BITS
 
 /* Define for large files, on AIX-style hosts. */
 #undef _LARGE_FILES
--- a/src/3rdparty/chromium/third_party/sqlite/src/sqlite_cfg.h.in
+++ b/src/3rdparty/chromium/third_party/sqlite/src/sqlite_cfg.h.in
@@ -126,6 +126,7 @@
 
 /* Number of bits in a file offset, on hosts where this is settable. */
 #undef _FILE_OFFSET_BITS
+#undef _TIME_BITS
 
 /* Define for large files, on AIX-style hosts. */
 #undef _LARGE_FILES
